plugins/linux-v4l2/v4l2-output.c

```diff
@@ -130,6 +174,13 @@ static bool try_connect(void *data, const char *device)
 	if (ioctl(vcam->device, VIDIOC_QUERYCAP, &capability) < 0)
 		goto fail_close_device;
 
+	if (!use_caps_workaround && !(capability.device_caps & V4L2_CAP_VIDEO_OUTPUT)) {
+		if (!try_reset_output_caps(device))
+			goto fail_close_device;
+		use_caps_workaround = true;
```

Comment:
To distinguish the `if` from subsequent code, either add a blank line after the `if` statement contents, or enclose it in braces.

Reply:
resolved in 26b447718